jib-core/src/main/java/com/google/cloud/tools/jib/global/JibSystemProperties.java

```diff
@@ -32,6 +32,8 @@
 
   private static final String SERIALIZE = "jibSerialize";
 
+  private static final String FORCE_LAYER_DOWNLOAD = "jib.forceDownload";
```

Comment:
`jib.cacheBase` with `always` or `as-needed`?

Reply:
Will work on this tomorrow. Need to update error messages that mention this property too. I think I'll throw `IllegalArgumentException` for wrong values.

Reply:
Will go for `jib.cacheBaseImage` (instead of `jib.cacheBase`) with acceptable values of `always` and `as-needed`.